src/etc/inc/ipsec.inc

```diff
@@ -1466,6 +1466,16 @@ function ipsec_setup_pools() {
 		return;
 	}
 	$scconf['pools']['mobile-pool'] = array();
+	$pool_addrs = array();
+	if (!empty($a_client['pool_address'])) {
+		$pool_addrs[] = "{$a_client['pool_address']}/{$a_client['pool_netbits']}";
+	}
+	if (!empty($a_client['pool_address_v6'])) {
+		$pool_addrs[] = "{$a_client['pool_address_v6']}/{$a_client['pool_netbits_v6']}";
+	}
+	if (count($pool_addrs)) {
+		$scconf['pools']['mobile-pool']['addrs'] = implode(',', $pool_addrs);
+	}
 	$pool_common =& $scconf['pools']['mobile-pool'];
```

Comment:
If you move `$pool_common =& $scconf['pools']['mobile-pool'];` above these changes, then when setting the pool `addrs` value you can do it this way:

    $pool_common['addrs'] = implode(',', $pool_addrs)

Which makes it more consistent with the way the rest of the settings are handled.

Reply:
ok, done